pkg/auth/oauth2/auth_openshift.go

```diff
@@ -33,6 +35,8 @@ type openShiftAuth struct {
 	k8sClient *http.Client
 
 	oauthEndpointCache *asynccache.AsyncCache[*oidcDiscovery]
+	sessions           *sessions.CombinedSessionStore
+	refreshLock        sync.Map
```

Comment:
noticed that we are missing the metrics field which is present in the `oicdAuth`. Not sure if thats on purpose

Reply:
I wasn't sure if we needed it here. We didn't have it before, so I thought probably not, since there were no metrics implemented in the OpenShift authenticator before.